src/Sulu/Component/Webspace/Manager/WebspaceManager.php

```diff
@@ -98,6 +98,34 @@ public function findPortalInformationByUrl(string $url, ?string $environment = n
         return null;
     }
 
+    public function findPortalInformationsByHost(
+        string $host,
+        bool $includeSubdomains = false,
```

Comment:
Is that parameter needed? I have never seen this function being called with a different value than `true`...

Reply:
Then I would rename the function, instead of introducing a basically unused parameter. How about `findPortalInformationsByHostIncludingSubdomains`? Would also make the calling code easier to understand, otherwise you always have to lookup what this boolean flag means...